Shippo/APIResource.cs

```diff
@@ -433,6 +433,23 @@ public ShippoCollection<Manifest> AllManifests (Hashtable parameters)
 
         #endregion
 
+        #region Track
+
+        public Track RetrieveTracking (String carrier, String id)
+        {
+            string ep = String.Format ("{0}/tracks/{1}/{2}", api_endpoint, carrier, id);
```

Comment:
Could you encode `carrier` and `id` in case they end up having reserved characters?